### PR TITLE
Closes #5

### LINKED ISSUE #5: def

### PR DESCRIPTION
**Pull request by rinqboun**
my pr

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+adam
```
Comment (AdamChecinski) [inline test.txt:1]: why did you create this file?

Comment (AdamChecinski) [inline test.txt:1]: abc

**Review by AdamChecinski — CHANGES_REQUESTED** [review]

File: test.txt (hunk 1 above)
Comment (rinqboun) [reply test.txt:1]: abc

**Comment by AdamChecinski** [comment]
nice

**Review by AdamChecinski — APPROVED** [review]
lgtm

**Pull request merged**